src/js/player.js

```diff
@@ -2430,6 +2437,15 @@ class Player extends Component {
     return this.cache_.currentTime;
   }
 
+  /**
+   * Apply the value of initTime stored in cache as currentTime.
+   *
+   * @private
+   */
+  applyInitTime_() {
+    this.techCall_('setCurrentTime', this.cache_.initTime);
```

Comment:
Maybe I should restore the value to zero after set the currentTime

Reply:
One alternative is to call `this.currentTime(this.cache_.initTime)`, then, there if get to the techCall_ line, we can unset (set back to 0) `initTime`, that way all the seeking related logic is always in `currentTime()`. Thoughts?

Reply:
agree, that way the for the `applyInitTime_`  method his only concern will be to get the time in the cache to call again the original logic at `currentTime`